api/src/pcapi/routes/backoffice_v3/templates/compliance/blacklist_domain_name_summary.html

```diff
@@ -0,0 +1,99 @@
+{% extends "layouts/connected.html" %}
+{% block page %}
+  <div class="pt-3 px-5">
+    <h2 class="fw-light">Règles de conformité &gt; Noms de domaines > Résumé</h2>
+    <div class="my-4">
+      {% if targeted_beneficiary_accounts | length == 0 %}
+        Aucune entrée ne correspond aux critères.
+      {% else %}
+        <div class="accordion accordion-flush"
+             id="blacklist-domain-names-accordion-summary">
+          <div class="accordion-item">
+            <h2 class="accordion-header">
+              <button class="accordion-button collapsed"
+                      type="button"
+                      data-bs-toggle="collapse"
+                      data-bs-target="#collapse-{{ domain_name | replace('.', '-') }}"
+                      aria-expanded="true"
+                      aria-controls="collapse-{{ domain_name | replace('.', '-') }}">
+                {% set count = targeted_beneficiary_accounts | length %}
+                {% if count > 1 %}
+                  <strong>{{ domain_name }}</strong>&nbsp;{{ count }} comptes seront désactivés
+                {% else %}
+                  <strong>{{ domain_name }}</strong>&nbsp;un compte sera désactivé
+                {% endif %}
+              </button>
+            </h2>
+            <div id="collapse-{{ domain_name | replace('.', '-') }}"
+                 class="accordion-collapse collapse"
+                 data-bs-parent="#blacklist-domain-names-accordion-summary">
+              <div class="accordion-body">
+                {% for user in targeted_beneficiary_accounts %}{{ user.email }} ({{ user.id }}),{% endfor %}
+              </div>
+            </div>
+          </div>
+        </div>
+      {% endif %}
+      {% if untouched_pro_accounts | length == 0 %}
+        <div class="alert alert-info mt-3"
+             role="alert">Aucune compte pro n'utilise ce nom de domaine</div>
+      {% else %}
+        <div class="alert alert-warning mt-3"
+             role="alert">
+          Voici les {{ untouched_pro_accounts | length }} comptes pros avec ce nom de domaine,
```

Comment:
```suggestion
          Voici les {{ untouched_pro_accounts | length }} comptes PRO avec ce nom de domaine,
```
pour être cohérent avec les lignes 58 et 60